module/retrieval/retrieval.go

```diff
@@ -249,6 +253,11 @@ func (q Auditor) Validate(ctx context.Context, validationInput module.Validation
 	}
 
 	results := make(map[Protocol]ResultContent)
+	/*
+	 * TODO:
+	 * totalBytes is total bytes for all protocols?
+	 *   or is this just supposed to be ResultContent.TotalBytes?
+	 */
 	totalBytes := uint64(0)
```

Comment:
totalBytes is total bytes for all protocols?

or is this just supposed to be ResultContent.TotalBytes?